charts/windows-ad-setup/templates/credentialspecs.yaml

```diff
@@ -4,7 +4,7 @@
 apiVersion: windows.k8s.io/v1
 kind: GMSACredentialSpec
 metadata:
-  name: {{ template "windows-ad-setup.name" $ }}-{{ $suffix }}
+  name: {{ $suffix }}

```

Comment:
This is just a fix to make sure we don't create the GMSACredentialSpec prefixed with `windows-ad-setup` for simplified reading. Change is correspondingly made in the Terraform module.